The build is broken

See https://app.circleci.com/pipelines/github/mozilla/addons?branch=main, and specifically https://app.circleci.com/pipelines/github/mozilla/addons/53/workflows/a5531d3c-2264-4ca8-803c-e9124933ef0b/jobs/53.

It looks like the build broke as of https://github.com/mozilla/addons/commit/1ebcf5792e824f97dcfcbb59cf66c2490d90416f, but there's nothing in that commit that should have caused it. Looking at the output, it seems like it probably actually broke as a result of https://github.com/mozilla/addons/commit/2a32da2a499509c49b89a57985dfe07258c5bf91

It doesn't seem to stop the docs from being built, though. I can see that the latest date on https://addons.readthedocs.io/en/latest/index.html is 2021-03-04T19:31:31Z. It seems like a warning from Sphynx is being treated as an error:

```
Warning, treated as error:
/home/circleci/addons/docs/server/push-duty.rst:155:Block quote ends without a blank line; unexpected unindent.
make: *** [Makefile:55: html] Error 2
```

Would be nice to have the build green, though.

--- a/docs/server/push-duty.rst
+++ b/docs/server/push-duty.rst
@@ -152,6 +152,7 @@ Once you are done pushing the tags to stage:
 
  * Run the ``./bin/run-l10n-extraction`` command in ``addons-frontend`` repository (`documentation <https://github.com/mozilla/addons-frontend/blob/master/docs/i18n.md>`_).
  * Run the ``./scripts/extract-l10n.sh`` command in ``addons-server`` repository.
+
 Because the `addons-server` is meant to be used inside the docker container, it doesn't have access to your `git` credentials, so you'll need to enter the container, run the script, exit the container and push the branch created by script. The `addons-frontend` one does that for you, but in both cases you'll need to open the pull request yourself.
 
 Before the push

